tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
```

Comment:
Using a composite project to speed up the build. Don't know how it works if we add TurboRepo later.